src/styles/move-control.scss

```diff
@@ -0,0 +1,67 @@
+.move-control {
```

Comment:
Hi @mogavin, Could you please put the copyright holder and license inside this file?

Reply:
Done !